meta-mender-raspberrypi/recipes-bsp/rpi-u-boot-scr/files/boot.cmd.in

```diff
@@ -2,5 +2,5 @@ fdt addr ${fdt_addr} && fdt get value bootargs /chosen bootargs
 run mender_setup
 mmc dev ${mender_uboot_dev}
 load ${mender_uboot_root} ${kernel_addr_r} /boot/uImage
-bootm ${kernel_addr_r} - ${fdt_addr}
+@@KERNEL_BOOTCMD@@ ${kernel_addr_r} - ${fdt_addr}
```

Comment:
This more specifically. Have not gotten hang of the review feature on github :)

Reply:
Ah, thanks! I'll remove it.